package-lock.json

```diff
@@ -2,11 +2,821 @@
   "requires": true,
   "lockfileVersion": 1,
   "dependencies": {
-    "prettier": {
-      "version": "1.18.2",
-      "resolved": "https://registry.npmjs.org/prettier/-/prettier-1.18.2.tgz",
-      "integrity": "sha512-OeHeMc0JhFE9idD4ZdtNibzY0+TPHSpSSb9h8FqtP+YnoZZ1sl8Vc9b1sasjfymH3SonAF4QcA2+mzHPhMvIiw==",
-      "dev": true
+    "@babel/code-frame": {
```

Comment:
this file should not have changed as you did not update the root file `package.json`

Reply:
fixed package-lock.json file in root